scripts/windows/msys2_prepare.bat

```diff
@@ -128,6 +128,9 @@ IF EXIST "%WIN_STYRENE_PATH%" IF "%FORCE_STYRENE_REINSTALL%" NEQ "true" GOTO STY
 %BASH_BIN% -lc "cd \"${ORIGINAL_TEMP}/styrene\" && %PIP% install ."
 :STYRENEINSTALLED
 
+@echo Patching request
+%BASH_BIN% -lc "sed -i -e 's/assert minor <= 24/assert minor <= 25/' C:/msys64/mingw64/lib/python3.7/site-packages/requests/__init__.py"
```

Comment:
🙀

Reply:
Move along nothing to see, don't worry I'm a professional ;-)